src/popup/popup.html

```diff
@@ -90,7 +90,9 @@
     <div id="snackbar-bookmarks" class="snackbar"></div>
     <div class="section-bookmarks">
       <div class="section-bookmarks--header">
-        <h2>Bookmarks</h2>
+        <div class="text-align-center vocab heading">
```

Comment:
Add the class to the heading element, not to the div.
Also, as your screenshot shows, the current size of heading is too small, please adjust it accordingly.